src/compiler/scala/tools/nsc/typechecker/TypeDiagnostics.scala

```diff
@@ -640,6 +640,12 @@ trait TypeDiagnostics {
       warnUnusedPatVars || warnUnusedPrivates || warnUnusedLocals || warnUnusedParams
     }
 
+    object Specials {
+      val Predef_=:= = typeOf[scala.Predef.=:=[_,_]].typeSymbol
+      val Predef_<:< = typeOf[scala.Predef.<:<[_,_]].typeSymbol
+      val Predef_Dummy = typeOf[scala.Predef.DummyImplicit].typeSymbol
```

Comment:
should these live in `[Run]Definitions`?

Reply:
Apparently. I got an error from sbt incremental compilation trying to use it locally.

Reply:
Actually I think I had an old compiler in the local ivy.

Reply:
Hmmm, yeah, I *think* `typeOf` is from the wrong universe. We should be looking at the compiler's compilation classpath, not the run-time one. I didn't investigate, but that's my intuition. Let's stick with the helpers in currentRun.runDefinitions. (Also note that this trait is mixed into analyzer, so any fields added here, add to its heft.)

Reply:
I thought I was being conservative. I guess I don't make a good conservative.